src/VisualStudio/Core/Next/ServicesVisualStudio.Next.csproj

```diff
@@ -82,6 +82,8 @@
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
     <PackageReference Include="Microsoft.Tpl.Dataflow" Version="$(MicrosoftTplDataflowVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Diagnostics.PerformanceProvider" Version="$(MicrosoftVisualStudioDiagnosticsPerformanceProviderVersion)" />
+    <PackageReference Include="StreamJsonRpc" Version="$(StreamJsonRpcVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="$(MicrosoftVisualStudioThreadingVersion)" />
```

Comment:
Yeah this is a weird one. The problem here is that MSBuild ends up seeing two MS.VS.Threading assemblies with different **assembly** versions. 

- The version which flows from the NuGet package referenced by ServicesVisualStudio.csproj
- The version which is referenced from other references MSBuild considers (a reference of a reference) 

The MSBuild assembly resolution logic considers these to be equal and hence issues a warning. Adding the explicit NuGet reference here fixes the problem because it wins